charts/prometheus-node-exporter/values.yaml

```diff
@@ -21,7 +21,6 @@ prometheus:
     enabled: false
     additionalLabels: {}
     namespace: ""
-    scheme: http
```

Comment:
why are we changing an existin value?
this would need to raise major version as this is a breaking change for updating users.

Reply:
Hmm, I was trying to avoid requiring specifying the same information in two places when configuring scheme. Perhaps it is best to leave monitor configuration with just the probe configuration added?

Reply:
Can't we just use the old var?